test/SampleApplication/.vscode/launch.json

```diff
@@ -7,10 +7,9 @@
 			"type": "reactnative",
 			"request": "launch",
 			"platform": "android",
-            "runtimeArgs": [ "--nolazy" ],
-            "cwd": "${workspaceRoot}",
+			"internalDebuggerPort": 9090,
 			"sourceMaps": true,
-			"outDir": "${workspaceRoot}/.vscode/"
+			"outDir": "${workspaceRoot}/.vscode/.react"
```

Comment:
It was accidentally, but I think we actually do want these updates, so I'm leaving this file as-is.